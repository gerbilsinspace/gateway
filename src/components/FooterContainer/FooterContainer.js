import React from 'react';
import Footer from '../Footer';

const FooterContainer = () => (
  <Footer />
);

export default FooterContainer;
