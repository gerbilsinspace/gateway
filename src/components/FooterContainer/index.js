import React from 'react';

const FooterContainer = () => (
  <footer />
);

export default FooterContainer;
