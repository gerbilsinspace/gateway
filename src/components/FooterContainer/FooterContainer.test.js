import React from 'react';
import { shallow } from 'enzyme';
import FooterContainer from './index';

let component;

describe('FooterContainer', () => {
  beforeEach(() => {
    component = shallow(<FooterContainer />);
  });

  it('should render without crashing', () => {
    expect(component).toBeDefined();
  });

  it('should match snapshot with no data', () => {
    expect(component).toMatchSnapshot();
  });
})