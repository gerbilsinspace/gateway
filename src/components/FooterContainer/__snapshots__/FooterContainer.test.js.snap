// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterContainer should match snapshot with no data 1`] = `<styled.footer />`;
